<html>
<head>
<title>TestNG:  Customer Guarantor Test</title>
<link href="../testng.css" rel="stylesheet" type="text/css" />
<link href="../my-testng.css" rel="stylesheet" type="text/css" />

<style type="text/css">
.log { display: none;} 
.stack-trace { display: none;} 
</style>
<script type="text/javascript">
<!--
function flip(e) {
  current = e.style.display;
  if (current == 'block') {
    e.style.display = 'none';
    return 0;
  }
  else {
    e.style.display = 'block';
    return 1;
  }
}

function toggleBox(szDivId, elem, msg1, msg2)
{
  var res = -1;  if (document.getElementById) {
    res = flip(document.getElementById(szDivId));
  }
  else if (document.all) {
    // this is the way old msie versions work
    res = flip(document.all[szDivId]);
  }
  if(elem) {
    if(res == 0) elem.innerHTML = msg1; else elem.innerHTML = msg2;
  }

}

function toggleAllBoxes() {
  if (document.getElementsByTagName) {
    d = document.getElementsByTagName('div');
    for (i = 0; i < d.length; i++) {
      if (d[i].className == 'log') {
        flip(d[i]);
      }
    }
  }
}

// -->
</script>

</head>
<body>
<h2 align='center'>Customer Guarantor Test</h2><table border='1' align="center">
<tr>
<td>Tests passed/Failed/Skipped:</td><td>0/0/1</td>
</tr><tr>
<td>Started on:</td><td>Wed Dec 27 09:16:29 COT 2023</td>
</tr>
<tr><td>Total time:</td><td>0 seconds (30 ms)</td>
</tr><tr>
<td>Included groups:</td><td></td>
</tr><tr>
<td>Excluded groups:</td><td></td>
</tr>
</table><p/>
<small><i>(Hover the method name to see the test class name)</i></small><p/>
<table width='100%' border='1' class='invocation-failed'>
<tr><td colspan='4' align='center'><b>FAILED CONFIGURATIONS</b></td></tr>
<tr><td><b>Test method</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='org.meto.appium.tests.TFCustomerGuarantor.setup()'><b>setup</b><br>Test class: org.meto.appium.tests.TFCustomerGuarantor<br>Parameters: org.testng.TestRunner@27305e6</td>
<td><div><pre>io.appium.java_client.service.local.AppiumServerHasNotBeenStartedLocallyException: The local appium server has not been started
Reason: Timed out waiting for [http://127.0.0.1:4723/status] to be available after 20002 ms
Consider increasing the server startup timeout value (currently 20000ms)
Node.js executable path: C:\Program Files\nodejs\node.exe
Arguments: [C:\Users\meto\AppData\Roaming\npm\node_modules\appium\build\lib\main.js, --port, 4723, --address, 127.0.0.1]
Output: [35m[Appium][39m Welcome to Appium v2.2.1
[35m[Appium][39m Non-default server args:
[35m[Appium][39m { address: [32m&apos;127.0.0.1&apos;[39m }
[35m[Appium][39m Attempting to load driver xcuitest...
[35m[Appium][39m Requiring driver at C:\Users\meto\.appium\node_modules\appium-xcuitest-driver

	at io.appium.java_client.service.local.AppiumDriverLocalService.start(AppiumDriverLocalService.java:203)
	at org.meto.appium.utils.AppiumUtils.startAppiumServer(AppiumUtils.java:65)
	at org.meto.appium.base.TestBase.setup(TestBase.java:44)
Caused by: org.openqa.selenium.net.UrlChecker$TimeoutException: Timed out waiting for [http://127.0.0.1:4723/status] to be available after 20002 ms
	at org.openqa.selenium.net.UrlChecker.waitUntilAvailable(UrlChecker.java:94)
	at io.appium.java_client.service.local.AppiumDriverLocalService.ping(AppiumDriverLocalService.java:157)
	at io.appium.java_client.service.local.AppiumDriverLocalService.start(AppiumDriverLocalService.java:182)
	... 21 more
Caused by: java.util.concurrent.TimeoutException
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:204)
	at org.openqa.selenium.net.UrlChecker.waitUntilAvailable(UrlChecker.java:91)
	... 23 more
... Removed 19 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1689924104", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1689924104'><pre>io.appium.java_client.service.local.AppiumServerHasNotBeenStartedLocallyException: The local appium server has not been started
Reason: Timed out waiting for [http://127.0.0.1:4723/status] to be available after 20002 ms
Consider increasing the server startup timeout value (currently 20000ms)
Node.js executable path: C:\Program Files\nodejs\node.exe
Arguments: [C:\Users\meto\AppData\Roaming\npm\node_modules\appium\build\lib\main.js, --port, 4723, --address, 127.0.0.1]
Output: [35m[Appium][39m Welcome to Appium v2.2.1
[35m[Appium][39m Non-default server args:
[35m[Appium][39m { address: [32m&apos;127.0.0.1&apos;[39m }
[35m[Appium][39m Attempting to load driver xcuitest...
[35m[Appium][39m Requiring driver at C:\Users\meto\.appium\node_modules\appium-xcuitest-driver

	at io.appium.java_client.service.local.AppiumDriverLocalService.start(AppiumDriverLocalService.java:203)
	at org.meto.appium.utils.AppiumUtils.startAppiumServer(AppiumUtils.java:65)
	at org.meto.appium.base.TestBase.setup(TestBase.java:44)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
	at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
	at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:317)
	at org.testng.SuiteRunner.run(SuiteRunner.java:276)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
	at org.testng.TestNG.runSuites(TestNG.java:1063)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: org.openqa.selenium.net.UrlChecker$TimeoutException: Timed out waiting for [http://127.0.0.1:4723/status] to be available after 20002 ms
	at org.openqa.selenium.net.UrlChecker.waitUntilAvailable(UrlChecker.java:94)
	at io.appium.java_client.service.local.AppiumDriverLocalService.ping(AppiumDriverLocalService.java:157)
	at io.appium.java_client.service.local.AppiumDriverLocalService.start(AppiumDriverLocalService.java:182)
	... 21 more
Caused by: java.util.concurrent.TimeoutException
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:204)
	at org.openqa.selenium.net.UrlChecker.waitUntilAvailable(UrlChecker.java:91)
	... 23 more
</pre></div></td>
<td>20</td>
<td>org.meto.appium.tests.TFCustomerGuarantor@2cc44ad</td></tr>
</table><p>
<table width='100%' border='1' class='invocation-skipped'>
<tr><td colspan='4' align='center'><b>SKIPPED CONFIGURATIONS</b></td></tr>
<tr><td><b>Test method</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='org.meto.appium.tests.TFCustomerGuarantor.beforeMethod()'><b>beforeMethod</b><br>Test class: org.meto.appium.tests.TFCustomerGuarantor</td>
<td></td>
<td>-1703686589</td>
<td>org.meto.appium.tests.TFCustomerGuarantor@2cc44ad</td></tr>
</table><p>
<table width='100%' border='1' class='invocation-skipped'>
<tr><td colspan='4' align='center'><b>SKIPPED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='org.meto.appium.tests.TFCustomerGuarantor.FillCustomerGuarantor()'><b>FillCustomerGuarantor</b><br>Test class: org.meto.appium.tests.TFCustomerGuarantor</td>
<td><div><pre>io.appium.java_client.service.local.AppiumServerHasNotBeenStartedLocallyException: The local appium server has not been started
Reason: Timed out waiting for [http://127.0.0.1:4723/status] to be available after 20002 ms
Consider increasing the server startup timeout value (currently 20000ms)
Node.js executable path: C:\Program Files\nodejs\node.exe
Arguments: [C:\Users\meto\AppData\Roaming\npm\node_modules\appium\build\lib\main.js, --port, 4723, --address, 127.0.0.1]
Output: [35m[Appium][39m Welcome to Appium v2.2.1
[35m[Appium][39m Non-default server args:
[35m[Appium][39m { address: [32m&apos;127.0.0.1&apos;[39m }
[35m[Appium][39m Attempting to load driver xcuitest...
[35m[Appium][39m Requiring driver at C:\Users\meto\.appium\node_modules\appium-xcuitest-driver

	at io.appium.java_client.service.local.AppiumDriverLocalService.start(AppiumDriverLocalService.java:203)
	at org.meto.appium.utils.AppiumUtils.startAppiumServer(AppiumUtils.java:65)
	at org.meto.appium.base.TestBase.setup(TestBase.java:44)
Caused by: org.openqa.selenium.net.UrlChecker$TimeoutException: Timed out waiting for [http://127.0.0.1:4723/status] to be available after 20002 ms
	at org.openqa.selenium.net.UrlChecker.waitUntilAvailable(UrlChecker.java:94)
	at io.appium.java_client.service.local.AppiumDriverLocalService.ping(AppiumDriverLocalService.java:157)
	at io.appium.java_client.service.local.AppiumDriverLocalService.start(AppiumDriverLocalService.java:182)
	... 21 more
Caused by: java.util.concurrent.TimeoutException
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:204)
	at org.openqa.selenium.net.UrlChecker.waitUntilAvailable(UrlChecker.java:91)
	... 23 more
... Removed 19 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace822409149", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace822409149'><pre>io.appium.java_client.service.local.AppiumServerHasNotBeenStartedLocallyException: The local appium server has not been started
Reason: Timed out waiting for [http://127.0.0.1:4723/status] to be available after 20002 ms
Consider increasing the server startup timeout value (currently 20000ms)
Node.js executable path: C:\Program Files\nodejs\node.exe
Arguments: [C:\Users\meto\AppData\Roaming\npm\node_modules\appium\build\lib\main.js, --port, 4723, --address, 127.0.0.1]
Output: [35m[Appium][39m Welcome to Appium v2.2.1
[35m[Appium][39m Non-default server args:
[35m[Appium][39m { address: [32m&apos;127.0.0.1&apos;[39m }
[35m[Appium][39m Attempting to load driver xcuitest...
[35m[Appium][39m Requiring driver at C:\Users\meto\.appium\node_modules\appium-xcuitest-driver

	at io.appium.java_client.service.local.AppiumDriverLocalService.start(AppiumDriverLocalService.java:203)
	at org.meto.appium.utils.AppiumUtils.startAppiumServer(AppiumUtils.java:65)
	at org.meto.appium.base.TestBase.setup(TestBase.java:44)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
	at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
	at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:317)
	at org.testng.SuiteRunner.run(SuiteRunner.java:276)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
	at org.testng.TestNG.runSuites(TestNG.java:1063)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: org.openqa.selenium.net.UrlChecker$TimeoutException: Timed out waiting for [http://127.0.0.1:4723/status] to be available after 20002 ms
	at org.openqa.selenium.net.UrlChecker.waitUntilAvailable(UrlChecker.java:94)
	at io.appium.java_client.service.local.AppiumDriverLocalService.ping(AppiumDriverLocalService.java:157)
	at io.appium.java_client.service.local.AppiumDriverLocalService.start(AppiumDriverLocalService.java:182)
	... 21 more
Caused by: java.util.concurrent.TimeoutException
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:204)
	at org.openqa.selenium.net.UrlChecker.waitUntilAvailable(UrlChecker.java:91)
	... 23 more
</pre></div></td>
<td>0</td>
<td>org.meto.appium.tests.TFCustomerGuarantor@2cc44ad</td></tr>
</table><p>
</body>
</html>